### DISCUSSION TITLE
A Simulation Score

### DISCUSSION THREAD

**Discussion by Python700** [MKT-Playbook/Hub#43] [Feature requests]
An idea I have is being able to simulate a score based off of the track and the DKG loadout. So, the process idea that I have for this is the user (1) select the track, (2) select the DKG and set Lv and Base Points for each, and (3) user submits and computes the estimated score. This way, the user would not need to adjust the Lv, Points on the card and forgets to change the stuff back. 

This can also help with the Reddit community, discord, etc. on how they can improve their scores and show them the Estimated Score of the track based on their loadouts.

**Comment by TitusPosiadly** [comment]
Hey, yes! Great idea. I had something similar in mind already and have some ideas that would fit in with this perfectly. #10 would also be a good fit for this tool. I will see if I can squeeze this in with the next bigger update 1.4. It is a rather simple tool so I might be able to. The most time would probably be taken up by the design. I will see what I can do. Thanks for the suggestion.
